snapshots/cardano-1.28.0.yaml

```diff
@@ -0,0 +1,175 @@
+name: cardano-1.28.0
+
+resolver: lts-17.6
+
+packages:
```

Comment:
I diffed with cardano-1.27.0.yaml and found that some of these packages are older than 1.28.0.

Reply:
Ah? It builds locally for me at least. Might not be the latest version of each package available though...

Reply:
Okay, I checked all packages one by one, removed those already in the LTS, and "virtually" separated them in two sections:

- Those not in the LTS
- Those in the LTS but with a different version (either older or more recent)

Few of them are present in LTS-18.3 (latest available), but not in LTS 17.15; so a good third can also be dropped when going for the next major lts.

Reply:
Oh, if there's LTS-18.x now, then let's use [LTS 18.4](https://www.stackage.org/lts-18.4)[

Also oops I reversed what I meant to say - some of the packages in this snapshot were older than those in `cardano-node-1.27.0.yaml`, but it's fixed now.